### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 22

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#322] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152323599-96cd84ef-c284-41e3-a4ea-6d922a9a9757.png)

**Comment by Exyss** [comment]
![Esercizio 22](https://user-images.githubusercontent.com/65043207/152661340-a58f7afc-0004-456c-954e-0db5167e202d.jpg)
